src/group.h

```diff
@@ -138,4 +138,7 @@ static void secp256k1_ge_storage_cmov(secp256k1_ge_storage *r, const secp256k1_g
 /** Rescale a jacobian point by b which must be non-zero. Constant-time. */
 static void secp256k1_gej_rescale(secp256k1_gej *r, const secp256k1_fe *b);
 
+/** Determine if a point (which is assumed to be on the curve) is in the group. */
```

Comment:
I think "is in group" is somewhat imprecise (which group?), even for readers who know about exhaustive tests. And others may really wonder what the point of this function is. 

The same is true for the name of the function. When I saw this, I first assumed it checks if the input is really on the point.

Reply:
Added some comments. WDYT?

Reply:
Yes, looks good! Now that you got a few more comments, I still think the function name can be improved. I don't have a great suggestion but something like  `ge_is_in_correct_subgroup`, `ge_is_in_subgroup`  would already make clear that this function is not about being on the curve or not. Note also the "is", I think that's more consistent with other names like `is_infinity` and makes clear that this is a getter.

Reply:
Done.